assets/src/edit-story/components/checklist/checks/elementLinkTappableRegionTooSmall.js

```diff
@@ -50,38 +56,38 @@ const ElementLinkTappableRegionTooSmall = () => {
     story,
     elementLinkTappableRegionTooSmall
   );
+  const setHighlights = useHighlights(({ setHighlights }) => setHighlights);
+
+  const { title, footer } = ACCESSIBILITY_COPY.linkTappableRegionTooSmall;
+
   return (
     elements.length > 0 && (
       <ChecklistCard
-        title={sprintf(
-          /* translators: %s: minimum tappable region size width x minimum tappable region size height. */
-          __('Increase tap area size to at least %s', 'web-stories'),
-          `${LINK_TAPPABLE_REGION_MIN_WIDTH}x${LINK_TAPPABLE_REGION_MIN_HEIGHT}px`
-        )}
-        /* titleProps={{ onClick: () => { perform highlight here } }} */
-        footer={
-          <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}>
-            {__(
-              'Make the linked element large enough for users to easily tap it',
-              'web-stories'
-            )}
-            {
-              //       <Link
-              //         href={'#' /* figure out what this links to */}
-              //         size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}
-              //       >
-              //         {'Learn more'}
-              //       </Link>
-            }
-          </Text>
+        title={title}
+        cardType={
+          elements.length > 1
+            ? CARD_TYPE.MULTIPLE_ISSUE
+            : CARD_TYPE.SINGLE_ISSUE
         }
-        /*
-        todo thumbnails for pages
+        footer={<DefaultFooterText>{footer}</DefaultFooterText>}
         thumbnailCount={elements.length}
-        thumbnail={<>
-            {elements.map(() => <Thumbnail />)}
-          </>}
-      */
+        thumbnail={
+          <>
+            {getVisibleThumbnails(elements).map((element) => (
+              <Thumbnail
+                key={element.id}
+                onClick={() => {
+                  setHighlights({
+                    elementId: element.id,
+                  });
+                }}
```

Comment:
🤔 this may not be necessary, but do we want to pull these into a callback so that they don't get re-created every render?

Reply:
yeah that's a good call, i'll fix that :)

Reply:
fixed!